shell/src/Luna/Shell/Command.hs

```diff
@@ -338,25 +346,29 @@ acquireGlobalConfig = liftIO $ Exception.catch acquire recovery where
 
     acquire :: IO (Either Text Global.Config)
     acquire  = do
-        homeDir <- Directory.getHomeDirectory
+        homeDir <- Path.getHomeDir
 
-        let lunaConfigDir = homeDir </> Global.configDir
-        configDirExists <- Directory.doesDirectoryExist lunaConfigDir
+        let configRelDirPath = Global.configDir
+
+        let lunaConfigDir =  homeDir </> configRelDirPath
+        configDirExists <- Path.doesDirExist lunaConfigDir
 
         unless configDirExists
-            $ Directory.createDirectoryIfMissing True lunaConfigDir
+            $ Path.createDirIfMissing True lunaConfigDir
+
+        let configNameRelFilePath = Global.configName
 
-        let lunaConfigFile = lunaConfigDir </> Global.configName
-        configFileExists <- Directory.doesFileExist lunaConfigFile
+        let lunaConfigFile = lunaConfigDir </> configNameRelFilePath
+        configFileExists <- Path.doesFileExist lunaConfigFile
 
         unless configFileExists $ do
             putStrLn "Generating Global Config"
-            putStrLn $ "Please fill in your name in " <> lunaConfigFile
-            putStrLn $ "Please fill in your email in " <> lunaConfigFile
+            putStrLn $ "Please fill in your name in " <> (Path.fromAbsFile lunaConfigFile)
```

Comment:
I'd replace the duplication here with a `let fpPath = Path.fromAbsFile lunaConfigFile` for cleanliness' sake.